webpack.config.js

```diff
@@ -47,7 +51,7 @@ module.exports = (env, argv) => ({ // eslint-disable-line no-undef
         rules: [
             {
                 test: /\.js$/,
-                exclude: /node_modules/,
+                exclude: /node_modules\/(?!(sulu-(.*)-bundle|@ckeditor|lodash-es)\/)/,
```

Comment:
Are these folders now correctly symlinked by both npm and yarn? Meaning that changes apply immediately when you are editing JS files from this repository?

Reply:
> But i don't see how this solves problems when moving files to assets/admin...

To the `glob` now NPM now that its a js file from external library defined in package.json and not a from the current project. So it resolves its requirements correctly thats why it solves problems moving files to assets/admin. `glob` should not be used to load things outside your project directory. and yes you are correctly we should remove glob from the requirements.

> Are these folders now correctly symlinked by both npm and yarn?  Meaning that changes apply immediately when you are editing JS files from this repository?

Yarn does never symlink this is just to let yarn work for the project where you don't change sulu/sulu code. Need to test if the watch task still works correctly for the core developer.